x-pack/functionbeat/scripts/mage/config.go

```diff
@@ -10,21 +10,13 @@ import (
 
 // XPackConfigFileParams returns the configuration of sample and reference configuration data.
 func XPackConfigFileParams() devtools.ConfigFileParams {
-	return devtools.ConfigFileParams{
-		ShortParts: []string{
-			devtools.OSSBeatDir("_meta/beat.yml"),
-			devtools.LibbeatDir("_meta/config.yml.tmpl"),
-		},
-		ReferenceParts: []string{
-			devtools.OSSBeatDir("_meta/beat.reference.yml"),
-			devtools.LibbeatDir("_meta/config.reference.yml.tmpl"),
-		},
-		ExtraVars: map[string]interface{}{
-			"ExcludeConsole":             true,
-			"ExcludeFileOutput":          true,
-			"ExcludeKafka":               true,
-			"ExcludeRedis":               true,
-			"UseDockerMetadataProcessor": false,
-		},
+	p := devtools.DefaultConfigFileParams()
+	p.Templates = append(p.Templates, "_meta/config/*.tmpl")
+	p.ExtraVars = map[string]interface{}{
+		"ExcludeConsole":             false,
+		"ExcludeFileOutput":          true,
+		"ExcludeKafka":               true,
+		"ExcludeRedis":               true,
+		"UseDockerMetadataProcessor": false,
```

Comment:
It looks like this file was changed in https://github.com/elastic/beats/pull/18148, which was only backported to 7.8. See https://github.com/elastic/beats/pull/18148/files#diff-e5fc36ab39d48c4f0f46ba9b287d4f38L13-L28. I think you need to do something like this:

```suggestion
	return devtools.ConfigFileParams{
		ShortParts: []string{
			devtools.OSSBeatDir("_meta/beat.yml"),
			devtools.LibbeatDir("_meta/config.yml.tmpl"),
		},
		ReferenceParts: []string{
			devtools.OSSBeatDir("_meta/beat.reference.yml"),
			devtools.LibbeatDir("_meta/config.reference.yml.tmpl"),
		},
		ExtraVars: map[string]interface{}{
			"ExcludeConsole":             false,
			"ExcludeFileOutput":          true,
			"ExcludeKafka":               true,
			"ExcludeRedis":               true,
			"UseDockerMetadataProcessor": false,
		},
```